[CN-1382]: Docs for Lite Members (#248)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/lite-members.adoc

== Configuring the Number of Lite Members

To configure the number of lite members you want to deploy in the Hazelcast CR, use the `count` field in the `liteMember` section of the Hazelcast CR.

.Example Lite Member Count Configuration
[source,yaml,subs="attributes+"]
----
include::ROOT:example$/lite-member.yaml[]
----

When you apply the given example, 3 data members and 2 lite members are going to be deployed.

== Configuring JVM Parameters

To configure JVM Parameters specifically for lite members, you can use the `jvm` field in the `liteMember` section of the Hazelcast CR.

.Example Lite Member JVM Configuration
[source,yaml,subs="attributes+"]
----
include::ROOT:example$/lite-member-jvm.yaml[]
----

== Configuring Environment Variables

To configure JVM Parameters specifically for lite members, you can use the `jvm` field in the `liteMember` section of the Hazelcast CR.

Same rules and logic is applicable as in the xref:env-vars.adoc[Environment variables].

== Configuring Resource Limits

To configure resource limits specifically for lite members, you can use the `resources` field in the `liteMember` section of the Hazelcast CR.

> .Example Lite Member Resource Limits Configuration

Reviewer: https://github.com/hazelcast/hazelcast-platform-operator/pull/1293/files#r1791588902

Author: https://github.com/hazelcast/hazelcast-platform-operator-docs/pull/248/commits/027cce4a0a2bde0f8c1738153e91ff2d4529ac4e

> = Deploying Lite Members

Reviewer: I think we should add some information at https://docs.hazelcast.com/operator/latest/connect-outside-kubernetes related to lite members. WDYT?

Author: Now we are testing service per pod, and if it works the same, I think we shouldn't add anything to this doc.

> You can the Hazelcast Platform Operator to deploy link:https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/lite-members[Lite Members]. You can use the separate `liteMember` section in the Hazelcast CR.

Reviewer 2: ```suggestion
Use the Hazelcast Platform Operator to deploy link:https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/lite-members[Lite Members]. You can use the separate `liteMember` section in the Hazelcast CR.
```

WARNING: When `jvm`, `env`, `resources`, `scheduling` sections are configured in the Hazelcast CR, it will not affect the lite member configuration. Lite members must be configured as explained in the next sections in this document.

== Configuring the Number of Lite Members

To configure the number of lite members you want to deploy in the Hazelcast CR, use the `count` field in the `liteMember` section of the Hazelcast CR.

.Example Lite Member Count Configuration
[source,yaml,subs="attributes+"]
----
include::ROOT:example$/lite-member.yaml[]
----

> When you apply the given example, 3 data members and 2 lite members are going to be deployed.

Reviewer 2: ```suggestion
When you apply the given example, 3 data members and 2 lite members are deployed.
```

> NOTE: As seen in the example, `clusterSize` stands for the number of data members to be created.

Reviewer 2: ```suggestion
NOTE: As seen in the example, `clusterSize` represents the number of data members to be created.
```
